qcfractal/interface/collections/dataset_view.py

```diff
@@ -162,8 +162,8 @@ def get_molecules(self, indexes: List[Union[ObjectId, int]]) -> List[Molecule]:
         with self._read_file() as f:
             mol_schema = f['molecule/schema']
             ret = [
-                Molecule(**self._deserialize_data(mol_schema[int(i) if isinstance(i, ObjectId) else i]))
-                for i in indexes
+                Molecule(**self._deserialize_data(mol_schema[int(i) if isinstance(i, ObjectId) else i]),
```

Comment:
Hows the new performance? I suspect it may not have changed much since the molecules should carry around a flag that they are validated.

Reply:
3.6s vs 4.3s for 10,000 molecules, so not a huge deal but still something.